include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2283,6 +2283,10 @@ BUILD_ASSERT(BT_GAP_SCAN_FAST_WINDOW == BT_GAP_SCAN_FAST_INTERVAL_MIN,
  *       In order to enable directed advertiser reports then
  *       @kconfig{CONFIG_BT_SCAN_WITH_IDENTITY} must be enabled.
  *
+ * @note Setting the `param.timeout` parameter is not supported when
+ *       @kconfig{CONFIG_BT_PRIVACY} is enabled. Supplying a nonzero timeout
+ *       will result in an -EINVAL error code.
```

Comment:
It only affects active scan, right? 

```suggestion
 * @note Setting the `param.timeout` parameter is not supported when
 *       @kconfig{CONFIG_BT_PRIVACY} is enabled when the param.type is @ref BT_LE_SCAN_TYPE_ACTIVE. Supplying a nonzero timeout
 *       will result in an -EINVAL error code.
```

Reply:
Yes, that's right.